components/applications-service/pkg/storage/postgres/hab_event.go

```diff
@@ -288,8 +224,13 @@ func (db *Postgres) upsertServiceAndMetadata(
 		updateStrategy = storage.NoneStrategy.String()
 	} else {
 		channel = svcMetadata.GetUpdateConfig().Channel
-		updateStrategy = svcMetadata.GetUpdateConfig().Strategy.String()
+		updateStrategy = storage.HabitatUpdateStrategyToStorageFormat(svcMetadata.UpdateConfig.GetStrategy()).String()
 	}
+
+	serviceGroupUniques := fmt.Sprintf("%s:%s:%s", svcMetadata.GetServiceGroup(), eventMetadata.GetApplication(), eventMetadata.GetEnvironment())
```

Comment:
🤔  Any ideas on how we might do better here? we could delimit with a different char that is more likely to be unused (like `;`)? Also, do we need to actually hash this value?

Reply:
Unicode! No I kid. Let's not do that. I am not sure if the colon will need to be escaped when it is in a url which is kind of gross, maybe actual hash would be better.